testing/run_all_tests_fp8.sh

```
@@ -0,0 +1,518 @@
+EVAL_BACKENDS=("vllm" "vllm" "vllm" "vllm")
+
+EVAL_BASE_DIR="./eval_results"
+MODEL_BASE_DIR="./regression_models"
+RESULTS_CSV="regression_results_fp8.csv"
+
+mkdir -p "$EVAL_BASE_DIR" "$MODEL_BASE_DIR"
+
+# ── Helper: activate environments ────────────────────────────────────────────
+
+activate_quant_env() {
+    source /home/HDCharles/rhdev/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: /home/HDCharles/rhdev/bin/activate was not specified as input (see shellcheck -x).

(SC1091)